--- mechanize/_mechanize.py	Thu Feb 12 01:14:25 2004
+++ mechanize/_mechanize.py	Thu Feb 12 01:15:07 2004
@@ -448,7 +448,7 @@
         self._links = []
         for token in p.tags(*(self.urltags.keys()+["base"])):
             if token.data == "base":
-                base = dict(token.attrs).get("uri")
+                base = dict(token.attrs).get("href")
                 continue
             if token.type == "endtag":
                 continue
